skiko/src/jvmMain/cpp/common/interop.cc

```diff
@@ -495,11 +523,11 @@ namespace skija {
             if (rectInts == nullptr)
                 return std::unique_ptr<SkIRect>(nullptr);
             else {
-                jint *ints = env->GetIntArrayElements(rectInts, NULL);
+                jint *ints = env->GetIntArrayElements(rectInts, 0);
```

Comment:
Signature is 
`jint * (JNICALL *GetIntArrayElements)(JNIEnv *env, jintArray array, jboolean *isCopy);` thus NULL or nullptr is better.